ansible/idrsystems-playbooks/idr-analysis.yml

```diff
@@ -34,3 +34,9 @@
   - role: openmicroscopy.versioncontrol-utils
 
   - role: openmicroscopy.celery-docker
+
+- hosts: idr-analysis
+  tasks:
+  - file:
+      path: /scratch
+      mode: "1777"
```

Comment:
Doesn't need to be `1777`. Given that we're using scratch on NFS I should probably have removed the partition it from the playbook in the first place.